packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx

```diff
@@ -46,43 +55,54 @@ export const accountLevelBadgeConfig: Record<
 
 export type AccountLevelBadgeProps = {
   badgeType: AccountLevelBadgeType
+  /* If truthy, show a button with the following props. If undefined, no button will be shown */
+  buttonProps?: ButtonProps
 }
 
 export const AccountLevelBadge: React.FunctionComponent<
   AccountLevelBadgeProps
-> = ({ badgeType }: AccountLevelBadgeProps) => {
+> = ({ badgeType, buttonProps }: AccountLevelBadgeProps) => {
   const badgeConfig = accountLevelBadgeConfig[badgeType]
   return (
     <Card
       sx={{
         maxWidth: 325,
         minWidth: 250,
         padding: '30px 20px',
-        display: 'flex',
-        width: '100%',
-        alignItems: 'center',
       }}
     >
-      <Tooltip title={badgeConfig.tooltipText}>
-        <Box alignSelf="start" mr="10px">
-          {badgeConfig.icon}
+      <Box display="flex" width="100%" alignItems="center">
```

Comment:
Move the flex layout to a new `Box` that captures all of the old content (icon, label, description)